src/utils/hooks/useSubmitReview.ts

```diff
@@ -33,8 +33,6 @@ const useSubmitReview: UseSubmitReview = (reviewId, reload) => {
 
     const reviewResponsesPatches = reviewResponses.map(({ thisReviewLatestResponse }) => ({
       patch: {
-        decision: thisReviewLatestResponse?.decision,
-        comment: thisReviewLatestResponse?.comment,
```

Comment:
You have asked this before. But that's on an older closed PR (remade due to trying to keep it alive)

The explanation is here: https://github.com/openmsupply/conforma-web-app/pull/1357#discussion_r996491751
> It isn't required now because we already do that for each review when the reviewer clicks on the "Submit" review button or on the "Agree with all". There was a problem with this code now which would overwrite the decision with "No decision" option.

Reply:
Okay, I didn't remember seeing the reply the first time. Thanks.